extensions/markdown/syntaxes/markdown.tmLanguage

```diff
@@ -566,11 +582,32 @@
 				<key>fenced_code_block_basic</key>
 				<dict>
 					<key>begin</key>
-						<string>(^|\G)\s*(([`~]){3,})\s*(html|htm|shtml|xhtml|inc|tmpl|tpl)(\s+.*)?$</string>
+					<string>(^|\G)\s*([`~]{3,})\s*(html|htm|shtml|xhtml|inc|tmpl|tpl)\n</string>
```

Comment:
Keep the `(\s+.*)?$`bit. We allow arbitrary text on the rest of the line after the language identifier to support passing other attributes (line line numbers) that some markdown engines support.